config/bidderinfo.go

```diff
@@ -605,6 +605,9 @@ func applyBidderInfoConfigOverrides(configBidderInfos BidderInfos, fsBidderInfos
 			if bidderInfo.EndpointCompression == "" && fsBidderCfg.EndpointCompression != "" {
 				bidderInfo.EndpointCompression = fsBidderCfg.EndpointCompression
 			}
+			if bidderInfo.OpenRTB == nil && fsBidderCfg.OpenRTB != nil {
+				bidderInfo.OpenRTB = fsBidderCfg.OpenRTB
+			}
```

Comment:
I believe there is purposefully no support for overriding the `OpenRTB` section because it describes how the adapter was programmed. No matter what override value is provided, the functionality of the adapter will not change - so why should we allow for overrides?